test/package-setup.test.ts

```diff
@@ -0,0 +1,53 @@
+import { describe, it, expect } from "vitest";
+import fs from "fs";
+
+describe("package.json changes should be explicit and tested in a template", () => {
+  it("Should be able to read the package.json file", () => {
```

Comment:
Instead of a snapshot, at some point using the template in a E2E test would be more valuable.